src/v2/mixins/mixins.js

```diff
@@ -1,17 +1,44 @@
 import { _ } from 'okta';
 
 _.mixin({
-  nestedOmit: function(obj, iteratee, context) {
-    var result = _.omit(obj, iteratee, context);
+  nestedOmit: function(obj, iteratee, context, dropObjectsByNameField) {
```

Comment:
I'm thinking something simpler (without extra `dropObjectsByNameField`)

- expect 2nd parameter is array. we don't need to be as generic as `_.omit` atm.
- adds `obj.name === "stateHandle"` is because drop by name is special case for `stateHandle`. But it's optional as it doesn't affect final result atm.

```
    nestedOmit: function(obj, names) {
    var result = _.omit(obj, names);

    if (names.includes(result.name) && result.name === "stateHandle") {
      result = _.omit(result, 'value');
    }

    _.each(result, function(val, key) {
      if (Array.isArray(val)) {
        result[key] = val.map((v) => {
          return _.nestedOmit(v, names);
        });
      } else if (typeof val === "object") {
        result[key] = _.nestedOmit(val, names);
      }
    });

    return result;
  },
```

Reply:
we can simplify this as @haishengwu-okta mentioned above ^^

Reply:
Used this approach but without  result.name === "stateHandle" check to omit value field
here ->
if (names.includes(result.name) && result.name === "stateHandle") {
      result = _.omit(result, 'value');
    }